src/examples/resources/js-source-config.json

```diff
@@ -2,24 +2,29 @@
   "source_converter_class_name": "io.synadia.flink.message.Utf8StringSourceConverter",
   "jetstream_subject_configurations": [
     {
-      "stream_name": "source-a",
-      "subject": "ua"
+      "stream_name": "source-b",
+      "subject": "ub1",
+      "ack_policy": "all"
```

Comment:
Please revert this file, but feel free to make another file with different policies.